emoji/emoji.go

```diff
@@ -0,0 +1,8 @@
+package emoji
+
+func getEmojiUnicode(emojiStr string) string {
```

Comment:
Do we need to export this?

Reply:
yes. we need to export this function. Updated.